core/shared/src/main/scala/scalaz/zio/IO.scala

```diff
@@ -0,0 +1,357 @@
+package scalaz.zio
+
+import scalaz.zio.Exit.Cause
+import scalaz.zio.internal.{ Executor, Platform }
+import scala.concurrent.ExecutionContext
+
+object IO {
```

Comment:
Can you please place all methods in alphabetical ordering for easy lookup / organization? Thank you!